azure-kusto-data/azure/kusto/data/_cloud_settings.py

```diff
@@ -18,43 +19,24 @@
 DEFAULT_PUBLIC_LOGIN_URL = "https://login.microsoftonline.com"
 DEFAULT_REDIRECT_URI = "https://microsoft/kustoclient"
 DEFAULT_KUSTO_SERVICE_RESOURCE_ID = "https://kusto.kusto.windows.net"
+DEFAULT_DEV_KUSTO_SERVICE_RESOURCE_ID = "https://kusto.dev.kusto.windows.net"

```

Comment:
Can we not get these from the Metadata endpoint?

Reply:
The whole point of these is to have a default when we are unable to get it from metadata (originally it was for clusters that didn't deploy the feature yet)

Reply:
Resolved offline